modules/react-components/src/components/popup/popup.tsx

```diff
@@ -27,7 +27,11 @@ import { PopupProps, Popup as SemanticPopup } from "semantic-ui-react";
  *
  * @returns
  */
-export const Popup = (props: PopupProps): ReactElement => {
+export const Popup: {
```

Comment:
Why do we change this file ?

Reply:
`PR Builder / ʦ Typecheck` was failing due to a type annotation error in this file as follows.

```
Error: ../../modules/react-components/src/components/popup/popup.tsx(30,14): error TS2742: The inferred type of 'Popup' cannot be named without a reference to 'apps/console/node_modules/semantic-ui-react'. This is likely not portable. A type annotation is necessary.
```